### ISSUE TITLE
support for epoch timestamps?

### ISSUE THREAD

**Issue by kjorg50** [LSIR/gsn#36]
Hello,

I am trying to write a virtual sensor using the [CSV Wrapper](https://github.com/LSIR/gsn/wiki/CSV-Wrapper), and one of the fields of my data is a timestamp in epoch format. I see that in CSVHandler.java you use the [`forPattern(String pattern)`](http://joda-time.sourceforge.net/api-release/org/joda/time/format/DateTimeFormat.html#forPattern%28java.lang.String%29) function to parse a date format given in the virtual sensor description XML file. For example:

``` xml
<predicate key="formats">numeric,timestamp("yyyy-MM-dd HH:mm:ss.SSS"),...</predicate>
```

However, the problem is that I don't know what pattern to use to allow for the input of an epoch time number such as 1318316400. Is it possible to use the epoch timestamp as a parameter to the DateTime() constructor, as in [this answer](http://stackoverflow.com/a/4142428)?

Basically, how can I use epoch time as an input for timestamps in GSN?

Thanks
